src/ui/views/ConfiguratorView/index.tsx

```diff
@@ -1212,6 +1212,18 @@ const ConfiguratorView: FunctionComponent<ConfiguratorViewProps> = (props) => {
                   {t('ConfiguratorView.Back')}
                 </Button>
 
+                {response?.buildFlashFirmware.success && (
+                  <Button
+                    sx={styles.button}
+                    color="primary"
+                    size="large"
+                    variant="contained"
+                    onClick={onFlash}
+                  >
+                    {t('ConfiguratorView.FlashAgain')}
```

Comment:
I think "Flash another" would sound more natural and would better align with describing the desired action.

Can you make this change?

Reply:
Yeah, it is done.